src/rdkafka.h

```diff
@@ -4763,6 +4765,11 @@ typedef struct rd_kafka_metadata {
 } rd_kafka_metadata_t;
 
 
+/**
+ * @brief Node information.
```

Comment:
Might be good to indicate that Node represents a broker, so:
`* @brief Node (broker) information.`

Reply:
done